ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/SandboxMain.scala

```diff
@@ -5,16 +5,31 @@ package com.digitalasset.platform.sandbox
 
 import java.util.concurrent.atomic.AtomicBoolean
 
+import ch.qos.logback.classic.Level
+import org.slf4j.Logger
 import com.digitalasset.platform.sandbox.cli.Cli
 import org.slf4j.LoggerFactory
 
 import scala.util.control.NonFatal
 
 object SandboxMain extends App {
 
+  // Copied from language-support/scala/codegen/src/main/scala/com/digitalasset/codegen/Main.scala
+  private def setGlobalLogLevel(verbosity: Level): Unit = {
```

Comment:
Could you please move this method to the bottom of this file? Having it first makes the program logic a bit noisy.

Reply:
Done